src/chrome/content/rules/Ofo.so.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+	Non-functional hosts
+		Couldn't connect to server:
+			 - cms.ofo.so
+			 - mail.ofo.so
+
+		4xx client error:
+			 - common.ofo.so
+			 - img.ofo.so
+			 - one.ofo.so
```

Comment:
Sorry that I cannot find working URL(s) for `img` and `one` other than the one you have suggested. I tried Google `inurl:img.ofo.so`, `site:img.ofo.so` and even `img.ofo.so`, none of these give me a working URL. What kind of search query  did you use?